forc-plugins/forc-doc/src/render.rs

```diff
@@ -326,21 +406,37 @@ fn all_items_list(title: String, list_items: Vec<(String, String)>) -> Box<dyn R
 fn _module_index() -> Box<dyn RenderBox> {
     box_html! {}
 }
+/// Sidebar component
 fn sidebar(
+    module_depth: usize,
     location: String,
-    href: String, /* sidebar_items: Option<Vec<String>>, */
+    href: String, /* sidebar_items */
```

Comment:
```suggestion
    sidebar_items: String,
```
Maybe we can self-document here like so?

Reply:
Hm, I'm not sure I understand what you mean. I could be more explicit, and put a "TODO"? I left it there as a reminder to add the sidebar items list, currently it's just empty.

Reply:
Ahhh, I thought it described what the contents of the `href` was :) the TODO helps!